### PR TITLE
Basque translations

### LINKED ISSUE #38726: Missing translations for Basque (eu)
Hello,

There are some missing translation for Basque (eu). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.eu.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.eu.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.eu.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.eu.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.eu.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.eu.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by ikerib**
Fix #38726

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.eu.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please select a valid color.</source>
+                <target>Mesedez, hautatu baliozko kolore bat.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Mesedez, hautatu baliozko herrialde bat.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Mesedez, hautatu baliozko txanpon bat.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.eu.xlf]: Could you double check this? Does it say "coin" or "currency"?

File: src/Symfony/Component/Form/Resources/translations/validators.eu.xlf [hunk 2]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter an integer.</source>
+                <target>Mesedez, sartu zenbaki oso bat.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Mesedez, hautatu baliozko hizkuntza bat.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Mesedez, hautatu baliozko eskualde-konfigurazio bat.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.eu.xlf:75]: "Locale" is often difficult to translate. Google Translate says that this translate to "regional configuration", is this correct?
Is this the word you would use for "locale"?

**Review by Nyholm — COMMENTED** [review]
Awesome. You are the best!

I used Google Translate to try to review/confirm this. I have two comments, I might be very wrong but can you double check?

I also see that you accidentally included some other commits and targeted 5.x instead of 4.4, but is okey. You dont have to fix that. The person that merges the PR will fix with that later. Just answer my two questions below and I'll be super happy.

**Comment by Nyholm** [comment]
@cluster28 if you have time and are able to, could you please review this?

File: src/Symfony/Component/Form/Resources/translations/validators.eu.xlf (hunk 1 above)
Comment (ikerib) [reply src/Symfony/Component/Form/Resources/translations/validators.eu.xlf]: You can use "moneta" or "txanpon". I updated the translation.

File: src/Symfony/Component/Form/Resources/translations/validators.eu.xlf (hunk 2 above)
Comment (ikerib) [reply src/Symfony/Component/Form/Resources/translations/validators.eu.xlf:75]: yes, it is correct.

**Comment by wetak** [comment]
@olatzromeo can you check that form me?

**Comment by cluster28** [comment]
@olatzromeo could you check that form me? Thnak you.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ikerib.

**Pull request merged**